admin/class-yoast-notification-center.php

```diff
@@ -550,8 +584,34 @@ public function update_storage() {
 	 * @return array|Yoast_Notification[] Registered notifications.
 	 */
 	public function get_notifications() {
+		if ( $this->notifications ) {
+			return call_user_func_array( 'array_merge', $this->notifications );
```

Comment:
Please do `array_merge( $this->notifications );` and if you need to have a recursive merge you can do: `array_merge_recursive`

Reply:
I opted to change it to 
```php
array_merge( ...$this->notifications )
```
(See my other comment for the reasoning behind this 🙂).